package.json

```diff
@@ -8,6 +8,7 @@
     "lint": "vue-cli-service lint"
   },
   "dependencies": {
+    "bootstrap": "^5.1.3",
```

Comment:
undo these changes really quick, pull up terminal and try,
>  vue add @techiediaries/bootstrap

Then add some bootstrap classes to header to see if it works, and commit the results